src/global/signal_handler.cc

```diff
@@ -152,6 +152,28 @@ void install_standard_sighandlers(void)
 #include "common/Thread.h"
 #include <errno.h>
 
+void get_name_by_pid(pid_t pid, char *task_name)
+{
+  char proc_pid_path[PATH_MAX] = {0};
+  snprintf(proc_pid_path, PATH_MAX, "/proc/%d/cmdline", pid);
+  int file_fd = open(proc_pid_path, O_RDONLY);
+
+  if (file_fd < 0) {
+    derr << "Failure to open '" << proc_pid_path 
+    << "' error= " << file_fd << dendl;
```

Comment:
@tchaikov  :tea: Thank you sir.Done it! In before I thought all need keep two space each new line.